libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/areaChart/PluggableAreaChart.tsx

```diff
@@ -333,9 +335,15 @@ export class PluggableAreaChart extends PluggableBaseChart {
         return referencePoint.uiConfig?.buckets?.[BucketNames.VIEW]?.itemsLimit ?? MAX_CATEGORIES_COUNT;
     }
 
+    private getBucketMeasures(extendedReferencePoint: IExtendedReferencePoint) {
```

Comment:
this method in all implementations can take only bucket param similar to getFilteredMeasuresForStackedCharts. Not whole reference point. It would minimize risk of some miss-use as we discussed